packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+import { CounterHandle, GaugeHandle, MeasureHandle } from './Handle';
+import { Metric } from './Metric';
+
+/**
+ * Meter is an implementation of the {@link Meter} interface.
+ */
+export class Meter implements types.Meter {
+  /**
+   * Creates and returns a new {@link Measure}.
+   * @param name the name of the metric.
+   * @param [options] the metric options.
+   */
+  createMeasure(
+    name: string,
+    options?: types.MetricOptions
+  ): Metric<MeasureHandle> {
+    throw new Error('not implemented yet');
```

Comment:
Same comment on not having errors, even for "not implemented yet" - I could imagine at some point us having some unimplemented thing that a user actually calls in production code by mistake